sdk/mobilenetwork/Azure.ResourceManager.MobileNetwork/src/Custom/ArmMobileNetworkModelFactory.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Azure;
+using Azure.Core;
+using Azure.ResourceManager.MobileNetwork;
+using Azure.ResourceManager.Models;
+using Azure.ResourceManager.Resources.Models;
+
+namespace Azure.ResourceManager.MobileNetwork.Models
+{
+    public static partial class ArmMobileNetworkModelFactory
+{
+    /// <summary> Initializes a new instance of MobileNetworkPacketCaptureData. </summary>
+    /// <param name="id"> The id. </param>
+    /// <param name="name"> The name. </param>
+    /// <param name="resourceType"> The resourceType. </param>
+    /// <param name="systemData"> The systemData. </param>
+    /// <param name="provisioningState"> The provisioning state of the packet capture session resource. </param>
+    /// <param name="status"> The status of the packet capture session. </param>
+    /// <param name="reason"> The reason the current packet capture session state. </param>
+    /// <param name="captureStartOn"> The start time of the packet capture session. </param>
+    /// <param name="networkInterfaces"> List of network interfaces to capture on. </param>
+    /// <param name="bytesToCapturePerPacket"> Number of bytes captured per packet, the remaining bytes are truncated. The default "0" means the entire packet is captured. </param>
+    /// <param name="totalBytesPerSession"> Maximum size of the capture output. </param>
+    /// <param name="timeLimitInSeconds"> Maximum duration of the capture session in seconds. </param>
+    /// <returns> A new <see cref="MobileNetwork.MobileNetworkPacketCaptureData"/> instance for mocking. </returns>
+    public MobileNetworkPacketCaptureData MobileNetworkPacketCaptureData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, MobileNetworkProvisioningState? provisioningState, MobileNetworkPacketCaptureStatus? status, string reason, DateTimeOffset? captureStartOn, IEnumerable<string> networkInterfaces, long? bytesToCapturePerPacket, long? totalBytesPerSession, int? timeLimitInSeconds)// copy the signature from the old code, and please remove all default values for those parameters
+=> MobileNetworkPacketCaptureData(id, name, resourceType, systemData, provisioningState, status, reason, captureStartOn, networkInterfaces, bytesToCapturePerPacket, totalBytesPerSession, timeLimitInSeconds, outputFiles = default);
```

Comment:
Please do a format to this file. and I assume this cannot compile, it should be:
```suggestion
=> MobileNetworkPacketCaptureData(id, name, resourceType, systemData, provisioningState, status, reason, captureStartOn, networkInterfaces, bytesToCapturePerPacket, totalBytesPerSession, timeLimitInSeconds, outputFiles: default);
```